pkg/controller/realmadmin/express_enable.go

```diff
@@ -60,7 +60,7 @@ func (c *Controller) HandleEnableExpress() http.Handler {
 		currentRealm.CodeDuration = enxSettings.CodeDuration
 		currentRealm.LongCodeLength = enxSettings.LongCodeLength
 		currentRealm.LongCodeDuration = enxSettings.LongCodeDuration
-		currentRealm.SMSTextTemplate = "Your Exposure Notifications verification link: [enslink] Expires in [longexpires] hours (click for mobile device only)"
+		currentRealm.SMSTextTemplate = database.DefaultENXSMSTextTemplate
```

Comment:
Should this only update the template if the template is the default non-ENX SMS template?

Reply:
this is how this has always worked - this is a "reset" kind of moment.